app/src/main/java/com/concordium/wallet/ui/account/accountsoverview/AccountsOverviewViewModel.kt

```diff
@@ -204,6 +208,22 @@ class AccountsOverviewViewModel(application: Application) : AndroidViewModel(app
         }
     }
 
+    private suspend fun handleAccountCreation(notifyWaitingLiveData: Boolean) {
+        val allAccounts = accountRepository.getAll()
+        if (allAccounts.isNotEmpty() && allAccounts.any { it.transactionStatus == TransactionStatus.FINALIZED }) {
```

Comment:
If .any{} succeeds, it automatically means the list is not empty. isNotEmpty check can be removed